magma/spec/censor_spec.rb

```diff
@@ -51,6 +51,23 @@ def to_record_set(model, revisions)
     )
   end
 
+  it 'new record revisions not censored for users with restricted permissions' do
```

Comment:
Nit: the name of this test seems confusing ... because the assertion is that the revision is censored? Should the user have an `A` permissions and no censored reasons?

Reply:
It's a bit misnomer, the spec ensures that the censor still enforces the attribute restriction but not the record one (which was the only way I could get it to ensure that it's not because the user skipped the entire restriction from `restrict?`).  I'll update the name and add a comment.  

In other words, the test is set up this way to verify the underlying assumption: the user IS being restricted, but NOT because of the record level one.